pyvista/themes.py

```diff
@@ -0,0 +1,2071 @@
+"""API description for managing plotting theme parameters in pyvista.
+
+Examples
+--------
+Apply a built-in theme
+
+>>> import pyvista
+>>> pyvista.set_plot_theme('default')
+>>> pyvista.set_plot_theme('document')
+>>> pyvista.set_plot_theme('dark')
+>>> pyvista.set_plot_theme('paraview')
+
+Load a theme into pyvista
+
+>>> theme = pyvista.themes.DefaultTheme()
+>>> theme.save('my_theme.json')  # doctest:+SKIP
+>>> loaded_theme = pyvista.load_theme('my_theme.json')  # doctest:+SKIP
+
+Create a custom theme from the default theme and load it into
+pyvista.
+
+>>> my_theme = pyvista.themes.DefaultTheme()
+>>> my_theme.font.size = 20
+>>> my_theme.font.title_size = 40
+>>> my_theme.cmap = 'jet'
+...
```

Comment:
I'm getting "this suggestion is invalid because no changes were made to the code" error when trying to "commit suggestion" in the browser.  I'll implement this in a commit.

Reply:
Actually, looking at the context, I think it's helpful to separate "setting the theme values" and "setting the global theme" from the new theme.

Reply:
Yeah, I figured it could be a (half-)intentional separator. That's why I just didn't remove it myself :)